examples/firewall/README.md

```diff
@@ -0,0 +1,30 @@
+Firewall
+==
+The Firewall NF is an NF which drops/forwards packets based on LPM rules.
+
+Compilation and Execution
+--
+```
+cd examples
+make
+cd firewall
+./go.sh INSTANCE_ID -d SERVICE_ID -f RULES_FILE
+
+OR
+
+./go.sh INSTANCE_ID -d SERVICE_ID -f RULES_FILE -b 
+
+cd examples
+./start_nf firewall INSTANCE_ID -d SERVICE_ID -f RULES_FILE
+
+OR
+
+./start_nf firewall INSTANCE_ID -d SERVICE_ID -f RULES_FILE -b
```

Comment:
Instead of these, ORs (same as the above) follow this example https://github.com/sdnfv/openNetVM/blob/develop/examples/simple_forward/README.md .

We only want to showcase different ways to start the NF.